v2/api-validator/src/client/SecureClient.ts

```diff
@@ -228,7 +228,9 @@ export function createSecurityHeaders(
 
 function getRelativeUrl(url: string) {
   const parsedUrl = new URL(url);
-  return parsedUrl.pathname + parsedUrl.search;
+  const relativeUrl = parsedUrl.pathname + parsedUrl.search;
+  const prefix = config.getServerUrlPrefix();
+  return relativeUrl.replace(prefix, '');
```

Comment:
I prefer using `relativeUrl.slice(prefix.length)`

Reply:
Consider move it to some `path-util` file

Reply:
Thanks, move to a different file